### PR TITLE
quadcontrol: yaw control neccessary changes

### PR DESCRIPTION
**Pull request by niewim19**
JIRA: PP-58

## Description

 - moved target attitude set above `motorsControl()`
 - mma motors order rewritten
 - added `ANGLE_HOLD` option for `motorsControl` so that the set value is current value and we only react to sudden changes in it

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [ ] Tested by hand on: (list targets here).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing linter checks and tests passed.
- [ ] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

**Comment by niewim19** [comment]
I do not know why whole `static int quad_attParse(FILE *file)` was treated again as addition

File: quadcontrol/control.c [hunk 1]
```diff
@@ -42,6 +42,7 @@
 #define ANGLE_THRESHOLD (M_PI / 6)
 #define RAD2DEG         ((float)180.0 / M_PI)
 #define DEG2RAD         0.0174532925f
+#define ANGLE_HOLD      INT16_MAX
```
Comment (HBuczynski) [inline quadcontrol/control.c:45]: Why not `INT32_MAX`?

**Review by HBuczynski — COMMENTED** [review]
Please move changes in `mma` module into separate commit.

File: quadcontrol/control.c (hunk 1 above)
Comment (niewim19) [reply quadcontrol/control.c]: My mistake, changed to INT32_MAX. Thanks!

**Pull request merged**